### PR TITLE
Adjust using event loop

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: guide/en/tutorial/using-with-event-loop.md [hunk 1]
```diff
@@ -40,7 +40,16 @@ in the state of a service made by previous request may affect current request. M
...
 There are two ways dealing with it. First, you can avoid having state by making services stateless. Second, you can
-clean up services at the end of the request processing.
+clean up services at the end of the request processing. In this case, a state resetter will help you:
+
+```php
+initializeContext();
+while ($request = getRequest()) {
+   $response = process($request);
+   emit($response);
+   $container->get(\Yiisoft\Di\StateResetter::class)->reset(); // We should reset the state of such services every request.
```
Comment (xepozz) [inline guide/en/tutorial/using-with-event-loop.md:50]: What the reason to get the resetter every request?

Comment (rustamwin) [reply guide/en/tutorial/using-with-event-loop.md:50]: See https://github.com/yiisoft/di#resetting-services-state

Comment (samdark) [inline guide/en/tutorial/using-with-event-loop.md:50]: I think @xepozz meant the following:

```suggestion
$resetter = $container->get(\Yiisoft\Di\StateResetter::class);
while ($request = getRequest()) {
   $response = process($request);
   emit($response);
   $resetter->reset(); // We should reset the state of such services every request.
```

Comment (rustamwin) [reply guide/en/tutorial/using-with-event-loop.md:50]: Oh, got it.

Comment (rustamwin) [reply guide/en/tutorial/using-with-event-loop.md]: Fixed.

**Pull request merged**